wallet/src/Dscp/Wallet/KeyStorage.hs

```diff
@@ -28,11 +28,17 @@ data StorageAccount = StorageAccount
 deriveJSON defaultOptions ''Storage
 deriveJSON defaultOptions ''StorageAccount
 
-storageFileName :: FilePath
-storageFileName = "run/tmp/wallet.json"
+getDataDir :: IO FilePath
+getDataDir = do
+    dir <- getXdgDirectory XdgData "disciplina"
+    createDirectoryIfMissing True dir
+    return dir
```

Comment:
We have similar logic in `Dscp.Config.AppDir`. I wouldn't complain, but keeping hardcoded application name in two different places doesn't look good.

Reply:
There's very high probability of all this code going into ariadne and using its settings pretty soon

Reply:
(At least reuse `appName`)

Reply:
Meh, okay